server/services/file_search_operations.py

```diff
@@ -25,41 +24,49 @@ def extract_sections(file_path, search_terms, sections, specify_lines, use_total
         start_line = term_line_num[i-1]
         line_empty = 0
```

Comment:
Please add the search_term as follows:
 search_term = search_terms[0] # for now (we will need to make it generic later) 
 and pass it as the parameter value to the is_content_line() function.


Also, add the following as the header_pattern above the for-loop (line 22) and pass it as the parameter value to the is_content_line() function:
header_pattern = r'^\s*NO\s+LB\s+ZA\s+FRAG\s+MASS\s+X\s+Y\s+Z\s*$'